src/backer.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity >=0.4.24;
+
+contract LenderTokenLike {
+    function transferFrom(address, address, uint) public;
+    function mint(address, uint) public;
+    function approve(address usr, uint wad) public returns (bool);
+    function balanceOf(address usr) public returns (uint);
+}
+
+contract BackerFab {
+    function deploy(address tkn_, address collateral_,address backer, address lightswitch_) public returns (address) {
+        Backer lender = new Backer(tkn_, collateral_,backer );
+        lender.rely(msg.sender);
+        return address(lender);
+    }
+}
+
+contract Backer {
+    // --- Auth ---
+    mapping (address => uint) public wards;
+    function rely(address usr) public auth { wards[usr] = 1; }
+    function deny(address usr) public auth { wards[usr] = 0; }
+    modifier auth { require(wards[msg.sender] == 1); _; }
+
+    // --- Data ---
+    LenderTokenLike public tkn;
+    LenderTokenLike public collateral;
+
+    address private backer;
+
+    constructor (address tkn_, address collateral_, address backer_) public {
```

Comment:
I think it's better not to require backer_ at instantiation. It makes the deploy more complicated. 
```suggestion
    constructor (address tkn_, address collateral_) public {
```